app/models/campaign.rb

```diff
@@ -115,9 +115,7 @@ def validate_dates
   # Intercept Rails typecasting and add error if given value cannot be parsed into a date.
   def validate_date_attribute(date_type)
     value = send("#{date_type}_before_type_cast")
-    # rubocop:disable Rails/TimeZone
```

Comment:
The removal of this is causing the build to break. Locally, you need to run `rspec --rails` in order to include the Rails-specific cops (in which case, this comment won't itself be a violation).